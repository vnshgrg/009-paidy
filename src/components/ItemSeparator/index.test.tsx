import "react-native";
import { Text } from "react-native";
import React from "react";
import { ItemSeparator } from "./index";

import renderer from "react-test-renderer";

it("renders ItemSeparator correctly", () => {
  const tree = renderer.create(<ItemSeparator />).toJSON();
  expect(tree).toMatchSnapshot();
});
