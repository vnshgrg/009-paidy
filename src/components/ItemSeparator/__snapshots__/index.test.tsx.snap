// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ItemSeparator correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#ddd",
      "height": 0.5,
    }
  }
/>
`;
